### PR TITLE
.github/workflows: Do not exempt PRs with milestone

### PR DESCRIPTION
**Pull request by guits**
The stale bot doesn't close certain PRs, even if they have been inactive for several years, because almost all of these PRs have a milestone

### CODE REVIEW

**Comment by guits** [comment]
As per the current configuration, if for some reason one thinks a PR should never be stalled, they can use the 'pinned' label.

**Comment by guits** [comment]
jenkins test make check

**Review by dmick — APPROVED** [review]
Shrug?   I don't much care about this, but it seems like an OK change to me

**Comment by cbodley** [comment]
this caused a bunch of reef backports to be closed over the holiday. valid backports shouldn't be closed this way, and i don't think anyone wants to micromanage backport PRs to prevent stalebot from doing this

instead of relying on stalebot for branches other than main, maybe someone could occassionally find/close all of the PRs targetting branches that have reached end-of-life. but has this really become a problem for anyone?

**Comment by guits** [comment]
Hey @cbodley ,

I assumed that if this 'stale bot' was set up, it was meant to close PRs that have had no activity for a certain period of time. Since I noticed in the logs that it was no longer closing any PRs, I modified its configuration. If this causes any issues, feel free to adjust the configuration.

1/ Increase the delay?
2/ Check if it’s possible to completely exempt the currently supported branches (downside is that this is something additional we would have to maintain).
3/ Completely remove the stale bot?

Although I don’t necessarily have a strong opinion on the matter, my personal take, and this is just my view, is that in general, when I see a project with hundreds of open PRs, many of them inactive for several years, it can give off a 'negative impression' of the project.

In short, feel free to modify if needed 👍
